packages/base-zone/src/types.js

```diff
@@ -30,9 +30,9 @@ export {};
  * @typedef {object} Stores
  * @property {() => Stores} detached obtain store providers which are detached (the stores are anonymous rather than bound to `label` in the zone)
  * @property {(specimen: unknown) => boolean} isStorable return true if the specimen can be stored in the zone, whether as exo-object state or in a store
- * @property {<K extends Key, V extends Passable>(label: string, options?: StoreOptions) => MapStore<K, V>} mapStore provide a Map-like store named `label` in the zone
- * @property {<K extends Key>(label: string, options?: StoreOptions) => SetStore<K>} setStore provide a Set-like store named `label` in the zone
- * @property {<K extends Key, V extends Passable>(
+ * @property {(label: string, options?: StoreOptions) => MapStore<any, any>} mapStore provide a Map-like store named `label` in the zone
+ * @property {<K>(label: string, options?: StoreOptions) => SetStore<K>} setStore provide a Set-like store named `label` in the zone
+ * @property {<K, V>(
```

Comment:
is this an edit of an earlier edit in this PR?
That suggests reviewing by commit (which I'm doing) is not a good idea.

It also suggests squashing. is that the plan?

Reply:
Plan is not to squash. I could squash some if you advise.

Reply:
one idea: smush all the commits together and then split them up by package, in topological order. In each package-commit, describe the changes made to that package.

not that important, but it would be kinda ideal

Reply:
That's a good idea. I've already done a lot of commit grooming on this PR but I'll try to remember to try that next time